### PR TITLE
[MCC-505819] Travis PG11

### PR DESCRIPTION
**Pull request by prutkowski-mdsol**
This PR sets Travis to use PostgreSQL 11.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: bionic
```
Comment (DMcKinnon-mdsol) [inline .travis.yml]: What does this do?

**Comment by DMcKinnon-mdsol** [comment]
Can we configure the tests to run against _multiple_ psql versions since this gem will likely still support psql 9.x and 10.x?

**Comment by DMcKinnon-mdsol** [comment]
@prutkowski-mdsol ping ^^

**Comment by hstebbins-mdsol** [comment]
What's up with this PR?

File: .travis.yml [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
-    - postgresql-contrib-9.5
     - libcurl3
     - libcurl3-gnutls
     - libcurl4-openssl-dev
-  postgresql: 9.5
+
 bundler_args: --jobs=1 --retry=3
+
+before_install:
+  - "curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | sudo bash"
```
Comment (hstebbins-mdsol) [inline .travis.yml]: Why download fossa here?

Comment (prutkowski-mdsol) [reply .travis.yml]: Because... type-o. This got ported over from the Dalton .travis.yml by accident. Removing it now.

**Pull request merged**